scripts/rhc-worker-playbook.worker.in

```diff
@@ -1,4 +1,4 @@
-#!/usr/libexec/platform-python
+#!/usr/bin/env @PYTHON@
```

Comment:
hmm this won't work if `PYTHON` is an absolute path; `PYTHON` comes from the `Makefile`:
```make
PYTHON               ?= python3
```
If we were to specify it from the packaging as e.g. `PYTHON="%{__python}"` (which we should), then that absolute path would break here.

I think a potential way to avoid this would be to use something like:
```make
PYTHON               ?= python3
PYTHON_PATH          = $(shell which $(PYTHON))
```
and then using it here:
```bash
#!@PYTHON_PATH@
```

Reply:
Good idea. I refactored the commit, incorporating these changes.